python/ray/dashboard/modules/metrics/dashboards/train_dashboard_panels.py

```diff
@@ -0,0 +1,59 @@
+# flake8: noqa E501
+from ray.dashboard.modules.metrics.dashboards.common import (
+    DashboardConfig,
+    Panel,
+    Target,
+)
+
+TRAIN_GRAFANA_PANELS = [
+    # Ray Train Metrics (Worker)
+    Panel(
+        id=1,
+        title="Checkpoint Report Time",
+        description="Time taken to report a checkpoint to storage.",
+        unit="seconds",
+        targets=[
+            Target(
+                expr="sum(ray_train_report_total_blocked_time_s{{{global_filters}}}) by (ray_train_run_name, ray_train_worker_world_rank)",
+                legend="Run Name: {{ray_train_run_name}}, World Rank: {{ray_train_worker_world_rank}}",
+            )
+        ],
+        fill=0,
+        stack=False,
+    ),
+    # Ray Train Metrics (Controller)
+    Panel(
+        id=2,
+        title="Train Controller Operation Time",
```

Comment:
"Overhead" makes more sense to me. These 2 metrics are actually blocking overhead.